library/alloc/src/vec.rs

```diff
@@ -1456,9 +1456,9 @@ impl<T> Vec<T> {
     /// If `new_len` is less than `len`, the `Vec` is simply truncated.
     ///
     /// This method uses a closure to create new values on every push. If
-    /// you'd rather [`Clone`] a given value, use [`resize`]. If you want
-    /// to use the [`Default`] trait to generate values, you can pass
-    /// [`Default::default()`] as the second argument.
+    /// you'd rather [`Clone`] a given value, use [`Vec::resize`]. If you
+    /// want to use the [`Default`] trait to generate values, you can
+    /// pass [`Default::default`] as the second argument.
```

Comment:
```suggestion
    /// pass [`Default::default()`] as the second argument.
```

Reply:
I specifically removed that bracket because `Default::default` is passed instead of `Default::default` in the code.

Reply:
Ah good point. This looks good then.